src/framework/mpas_dmpar.F

```diff
@@ -349,7 +349,13 @@ subroutine mpas_dmpar_abort(dminfo)!{{{
 #ifdef _MPI
       integer :: mpi_ierr, mpi_errcode
 
-      call MPI_Abort(dminfo % comm, mpi_errcode, mpi_ierr)
+      !
+      ! Only if MPI was initialized by MPAS should we call MPI_Abort
+      ! Otherwise, MPI was initialized outside of MPAS and we should not abort on MPI_COMM_WORLD
+      !
+      if ( dminfo % initialized_mpi ) then
+         call MPI_Abort(dminfo % comm, mpi_errcode, mpi_ierr)
+      end if
```

Comment:
@jjguerrette These changes seem like they wouldn't be useful to JEDI, since even if we avoided calling `MPI_Abort` here, line 361 (just a few lines below) would execute a Fortran `stop` statement. I suppose these changes were made on the path to the changes in the `mpas_log.F` abort behavior. If that's the case, could you confirm that reverting these changes in `mpas_dmpar.F` would not be problematic for your use case?

Reply:
As discussed in person, this mpas_dmpar_abort may no longer be called.  We may need to revisit `mpas_dmpar_global_abort` when I have an appropriate use-case, but for now the changes to this subroutine can be reverted.